packages/manager/apps/pci-rancher/src/public/translations/pci-rancher/order/Messages_fr_FR.json

```diff
@@ -0,0 +1,14 @@
+{
+  "order_catalog_price_free": "Inclus",
+  "order_catalog_price_tax_excl_label": "{{ price }} HT",
+  "order_catalog_price_tax_incl_label": "{{ price }} TTC",
+  "order_catalog_price_gst_excl_label": "{{ price }} ex. GST",
+  "order_catalog_price_gst_incl_label": "{{ price }} incl. GST",
+  "order_catalog_price_interval_day": "jour",
+  "order_catalog_price_interval_month": "mois",
+  "order_catalog_price_interval_year": "an",
+  "order_catalog_price_interval_hour": "heure",
+  "order_catalog_price_from": "De",
+  "order_catalog_price_to": "à",
+  "order_catalog_price_beta_free": "Gratuit pendant la bêta"
+}
```

Comment:
This content seems to be duplicated a lot already on the Manager repo. Can yo please move it to reusable place to avoid duplicating?